texthero/representation.py

```diff
@@ -782,8 +802,12 @@ def dbscan(
     --------
     >>> import texthero as hero
     >>> import pandas as pd
-    >>> s = pd.Series(["Football, Sports, Soccer", "music, violin, orchestra", "football, fun, sports", "music, enjoy, guitar"])
-    >>> s = s.pipe(hero.clean).pipe(hero.tokenize).pipe(hero.tfidf).pipe(hero.flatten) # TODO: when others get Representation Support: remove flatten
+    >>> s = pd.Series(["Football, Sports, Soccer", "music, violin, orchestra",
+    ...                "football, fun, sports", "music, enjoy, guitar"])
+    >>> s = s.pipe(hero.clean) \
```

Comment:
Good catch!
````
s = (
   s.pipe(hero.clean)
   ...
    .pipe(hero.flatten)
)
```

might be even nicer? this is how we do it for instance in the README ...